src/renderer/services/midgard/pools.ts

```diff
@@ -104,13 +99,9 @@ const createPoolsService = (
                 FP.pipe(
                   // As midgard v2 is missing poolSlipAverage and swappingTxCount
                   // field we have to get them from getSwapHistory
+                  // Emulate ALL period of a pool life
```

Comment:
[question] Where do you emulate something? _Maybe this line ^ is just an artifact and can be removed...?_

Reply:
yes it can be removed, so if we don't set `from`, `to` in the query, it assumes `all time` by default.